lib/teleterm/daemon/daemon.go

```diff
@@ -1283,6 +1284,14 @@ func (s *Service) ClearCachedClientsForRoot(clusterURI uri.ResourceURI) error {
 	return trace.Wrap(s.clientCache.ClearForRoot(profileName))
 }
 
+// ClearCachedStaleClientsForRoot closes and removes clients from the cache
+// for the root cluster and its leaf clusters, if their cert is outdated.
+func (s *Service) ClearCachedStaleClientsForRoot(clusterURI uri.ResourceURI) error {
```

Comment:
I think this should be `ClearStaleCachedClientsForRoot` since "stale" is an adjective here and "cached" describes a type of a client.